endpoints/globus/endpoint_test.go

```diff
@@ -78,8 +79,17 @@ endpoints:
       client_secret: ${DTS_GLOBUS_CLIENT_SECRET}
 `, sourceEndpointName, sourceEndpointId)
 
+func enableDebugLogging() {
+	logLevel := new(slog.LevelVar)
+	logLevel.Set(slog.LevelDebug)
+	h := slog.NewJSONHandler(os.Stderr, &slog.HandlerOptions{Level: logLevel})
+	slog.SetDefault(slog.New(h))
+}
+
```

Comment:
this code snippet looks like it appears a couple of times -- worth extracting out into a utils-type package?

Reply:
I had considered a `testing` or `testutils` package for this reason, but I didn't act on it. Seems like it's probably worth it, though.

Reply:
I added a `dtstest` package with this function in it (capped, since only functions and variables with capitalized names are exported from packages. I didn't really like this until I started using it, and it's actually a really great mechanism for controlling the interface!).